### DISCUSSION TITLE
开学了，呜呜呜

### DISCUSSION THREAD

**Discussion by nizhimin** [share121/inter-knot#4751] [绝区零]
请问各位假期过得怎么样，本人不怎么良好

**Comment by XingYeWeiH** [comment]
进厂了X﹏X

**Comment by Robin-txdy** [comment]
还不错， 只是有点可惜没去旅游<img src="https://img-static.mihoyo.com/communityweb/upload/6bdba3c03727d9e42ff5432d3e422751.png" alt="偶像姬-沮丧" width="75" >

**Comment by Hangover-Cyan-Eternally** [comment]
开学及放假，还有“工资”拿，哈哈

**Comment by QQZYZY** [comment]
你说你，不想在这里！我也不想在这里！
![Uploading mmexport1715686277943.jpg…]()

**Comment by askytaptap** [comment]
💊军训了😭
